### DISCUSSION TITLE
Scraping weirdness with "The Twelve Kingdoms"

### DISCUSSION THREAD

**Discussion by mihawk90** [Komet/MediaElch#1690] [Q&A] [answered]
So this is the first time I stumbled on this and I post it in discussions because I don't believe it's necessarily a bug, I'm just wondering what's going on or if there's a way to fix it on my side.

This is about an anime called "The Twelve Kingdoms":
https://www.themoviedb.org/tv/42729-the-twelve-kingdoms

When I scrape the entire series, for whatever reason it's not getting any info for the 3rd season. However, when I scrape each episode of Season 3 manually, it works just fine:

![image](https://github.com/Komet/MediaElch/assets/556162/a28769f5-5313-41c1-94e7-f87d508d139d) ![image](https://github.com/Komet/MediaElch/assets/556162/df5854e4-cf6e-4047-a370-d755a64942b6)

Don't mind the Season 1440 at the bottom, it's just a mismatch from filenames in a subdirectory.

Are there any logs I can pull to see what's going on? Evidently the data is returned by the API in some way, I'm just wondering if it's missing in the series-info-API for whatever reason or if it's ME having a stroke.

And yeah I tested it a couple times, it's consistent, not a one-off weirdness.

Any idea why could be going on? :thinking:

**Comment by bugwelle** [comment, marked as answer]
Oh! I have an idea. It could be that we're not using pagination of TMDb (there may be a TODO comment somewhere :facepalm: )

Thanks for bringing this to my attention! I'm making this discussion an issue. :)

=> https://github.com/Komet/MediaElch/issues/1692

**Reply by mihawk90** [reply]
Thanks, odd that Season 4 is scraped properly then :thinking:

**Reply by bugwelle** [reply]
Well, was just my assumption. I'll need to look into it :D

**Reply by bugwelle** [reply]
Pagination isn't done for TMDB, as TMDB doesn't provide it. I looked into this issue and sadly couldn't reproduce it. If it still persists, please let me know. Please see https://github.com/Komet/MediaElch/issues/1710 for notes on why MediaElch development has slowed done.
